app/services/analytics.rb

```diff
@@ -23,8 +23,8 @@ def request_attributes
     }
   end
 
-  def ahoy
-    @ahoy ||= Rails.env.test? ? FakeAhoyTracker.new : Ahoy::Tracker.new(request: request)
+  def analytics
+    @analytics ||= Rails.env.test? ? FakeKeen : PublishAnalyticsJob
```

Comment:
Rather than leak the `FakeKeen` into "production" code, what if we did a `stub_const("PublishAnalyticsJob", FakeKeen)` in tests?